client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx

```diff
@@ -78,6 +81,20 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
                                                     label='אין מענה'
                                                 />
                                             </Grid>
+                                            <Grid container item xs={2}>
+                                                <Tooltip title={interactedContact.phoneNumber !== null ? callContact : noPhoneNumber}>
+                                                    <span>
+                                                        <IconButton
+                                                            color='primary'
```

Comment:
phone icon button already exists in investigation header. consider making it generic and using it?

Reply:
I considered it, but here it used with span to display tooltip when disabled.